src/components/collection/Selector/Search.tsx

```diff
@@ -0,0 +1,139 @@
+import {
+    Autocomplete,
+    AutocompleteChangeReason,
+    autocompleteClasses,
+    Box,
+    Popper,
+    Skeleton,
+    TextField,
+} from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { isEqual } from 'lodash';
+import { useState } from 'react';
+import { useIntl } from 'react-intl';
+import {
+    detectAutoCompleteInputReset,
+    detectRemoveOptionWithBackspace,
+} from 'utils/mui-utils';
+import { CollectionData } from './types';
+import ListboxComponent from './VirtualizedList';
+
+interface Props {
+    options: any[];
+    onChange: (collections: string[], reason: AutocompleteChangeReason) => void;
+    selectedCollections: string[] | CollectionData[];
+    readOnly?: boolean;
+    getValue?: (option: any) => string;
+    AutocompleteProps?: any; // TODO (typing) - need to typ as props
+}
+
+const StyledPopper = styled(Popper)({
+    [`& .${autocompleteClasses.listbox}`]: {
+        'boxSizing': 'border-box',
+        '& ul': {
+            padding: 0,
+            margin: 0,
+        },
+    },
+});
+
+function CollectionSelectorSearch({
```

Comment:
Moving search into stand alone. Will be especially helpful when we build out the search to be more than a standard autocomplete.